include/rsp_dma.inc

```diff
@@ -120,6 +120,7 @@ DMAInEnd:
     andi $1, s0, 7
     add s4, $1
     andi t2, 0xF   # isolate DMA wait flags
+SpStatusWait:
```

Comment:
Why not use `DMAWaitLoop`?

Reply:
DMAWaitLoop is a misnomer, because the code just waits for a specific status bit to be set. In the context of rsp_dma.inc it is used to wait for DMA to be not full or idle, but we realized we could reuse the code for a generic "wait for a bit in SP status". Since we can't rename an existing label in a shared assembly library (as it's basically public API), we added a new label with a name that it's cleared in the context of rspq. If you read the code in rspq that uses it, it would have been highly confusing to see a call to DMAWaitLoop as that code has nothing to do with DMA.